Binding/ViewModel.cs

```diff
@@ -0,0 +1,32 @@
+using Godot.Community.ControlBinding.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using Godot.Community.ControlBinding.Extensions;
+
+namespace Godot.Community.ControlBinding;
+
+public abstract partial class NodeViewModel : Node, IViewModel
```

Comment:
Maybe you can implement something similar to: https://github.com/canton7/PropertyChanged.SourceGenerator 
So users will not need to extend from this two classes and they will not need to create the boilerplate  :)
```C#
private bool labelIsVisible = true;
    public bool IsAddNewPlayerEnabled
    {
        get { return labelIsVisible; }
        set { this.SetValue(ref labelIsVisible, value); }
    }
```

Reply:
It might even be possible for users to use it directly if we use INotifyPropertyChanged instead of a custom event handler. 

The custom event handler was originally required to support Godot Signals in C#, but after moving to raw events, we're no longer restricted.